bedevere/bpo.py

```diff
@@ -37,6 +40,19 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
+            body = event.data["pull_request"]["body"]
+            if CLOSING_TAG not in body:
+                issue_number = issue_number_found.group("issue")
+                BPO_MSG = textwrap.dedent(f"""\
```

Comment:
And it's merged, so once Travis is green for master again it will go live.

And I don't know why textwrap.dedent() isn't working since I don't see how an f-string would influence the outcome in anyway. My guess is the string as being inserted has some unexpected whitespace that's throwing the algorithm off.